### DISCUSSION TITLE
Ability to block `/back` into forbidden worlds

### DISCUSSION THREAD

**Discussion by Baterka** [Zrips/CMI#6339] [Enhancement]
On our server, we have separate world for events. Players can join this world by warp. When user die inside this event world, he is respawned at his respawnpoint (home, bed, spawn, etc.).

Problem is that players with `/back` permission can go back into this event world and are relocated back on death position. This is not great for some usecases.

Nice enhancement of `/back` should be that if player does not have permission in destination world, he cannot `/back`.

**Comment by dkalaxdk** [comment]
Already a feature.
![image](https://user-images.githubusercontent.com/13720707/142733697-3c79d884-f8a6-4a44-a52a-00c16b170663.png)
